src/track/trackref.cpp

```diff
@@ -1,6 +1,50 @@
 #include "track/trackref.h"
 
 
+namespace {
+
+inline
+QUrl urlFromLocalFilePath(QString localFilePath) {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+    return QUrl::fromLocalFile(std::move(localFilePath));
+#else
+    if (localFilePath.isEmpty()) {
+        return QUrl();
+    } else {
+        return QUrl::fromLocalFile(std::move(localFilePath));
+    }
+#endif
+}
+
+inline
+QString uriFromUrl(const QUrl& url) {
```

Comment:
A URL is a subset of all URIs. Does the function name makes any sense? 

urlToString() or something?

Reply:
There are various ways to format a URL as a string. The semantic *URI* type uses special normalization and encoding rules to obtain a canonical string representation. Wrapping the resulting QString into a new type is not worth the overhead (at least not now).

Reply:
It is just about the name.
You cannot make a uri from a url, because a url is always also a uri.
We just need to find a new name for this function.

Sometimes we need the string presentation of an url using a type for this is IMHO not nessesary.

Reply:
I didn't consider that this is only a hidden utility function.